packages/broker/test/unit/ConfigWizard.test.ts

```diff
@@ -218,7 +207,6 @@ describe('ConfigWizard', () => {
             const importPrivateKey = Wallet.createRandom().privateKey
```

Comment:
The name of the variable? As it has the verb prefix in the name, it sounds like an action (similar to `generateOrImportPrivateKey`) and not the content itself